public/style.css

```
@@ -0,0 +1,70 @@
+body{
+    width: 600px;
+    margin: auto;
+    background-color: #E0E0E0;
+}
+a.top{
```

```suggestion
.top{
```

CSSはなるべくタグ名に依存しない方がよいです。（他のクラスも同様）